src/index.css

```diff
@@ -104,6 +105,70 @@ h1 {
   margin-top: 8px;
 }
 
+nav {
+  background-color: #262626;
+  width: 100%;
+  position: sticky;
+  top: 0;
+  height: auto;
+}
+
+nav li {
+  font-size: 19px;
+  font-weight: 700;
+  line-height: 26px;
+  text-align: center;
+  margin-right: 16px;
+  margin-bottom: 8px;
+  margin-left: 16px;
+  padding: 8px 16px 8px 16px;
+}
+
+nav li:hover {
+  background-color: #797876;
+}
+
+nav ul {
+  list-style-type: none;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  height: auto;
+  padding-bottom: 66px;
+  padding-top: 30px;
+}
+
+nav a {
+  text-decoration: none;
+  color: white;
+}
+
+nav a.active-link {
+  background-color: #ffffff;
+  color: black;
+}
+nav ul li:has(a.active-link) {
+  background-color: white;
+  color: black;
+}
+
+.menu-button {
+  font-family: "Material Symbols Outlined";
+  font-weight: normal;
+  font-style: normal;
+  font-size: 24px;
+  text-align: right;
+  padding: 16px;
+  color: #ffffff;
+}
```

Comment:
I checked the stylesheet that you're importing from Material Symbols and it already contains the class `.material-symbols-outlined`:

```css
.material-symbols-outlined {
  font-family: 'Material Symbols Outlined';
  font-weight: normal;
  font-style: normal;
  font-size: 24px;
  line-height: 1;
  letter-spacing: normal;
  text-transform: none;
  display: inline-block;
  white-space: nowrap;
  word-wrap: normal;
  direction: ltr;
  -webkit-font-feature-settings: 'liga';
  -webkit-font-smoothing: antialiased;
}
```

So, you don't need to create your own class. You can use `material-symbols-outlined` in your component instead of defining this `menu-button` class.

Or what were your reasons not to do that?